resources/views/users/partials/create-user-form.blade.php

```diff
@@ -0,0 +1,109 @@
+<section>
+    <form id="send-verification" method="post" action="{{ route('verification.send') }}">
+        @csrf
+    </form>
+
+    <form method="post" action="{{ route('users.store') }}" class="mt-6 space-y-6">
+        @csrf
+        @method('post')
+
+        <!-- Name -->
+        <div>
+            <div style="display: flex; direction: column">
+                <x-input-label for="name" :value="__('Name')" />
+                <x-required-asterisk/>
+            </div>
+            <x-text-input id="name" name="name" type="text" class="mt-1 block w-full" placeholder="Blue Folf" required autofocus autocomplete="name" />
+            <x-input-error class="mt-2" :messages="$errors->get('name')" />
+        </div>
+
+        <!-- Email -->
+        <div>
+            <div style="display: flex; direction: column">
+                <x-input-label for="email" :value="__('Email')" />
+                <x-required-asterisk/>
+            </div>
+            <x-text-input id="email" name="email" type="email" class="mt-1 block w-full" placeholder="user@domain.extension" required autocomplete="username" />
+            <x-input-error class="mt-2" :messages="$errors->get('email')" />
+        </div>
+
+        <!-- User is Active -->
+        <div>
+            <x-input-label for="active" :value="__('Active Status')" />
+            <x-select-input id="active" name="active" class="block text-sm" required>
+                <option value="1" {{ old('active', $user->active) == 1 ? 'selected' : '' }}>Active</option>
+                <option value="0" {{ old('active', $user->active) == 0 ? 'selected' : '' }}>Inactive</option>
+            </x-select-input>
+            <x-form-validation for="active" />
+            <x-input-error class="mt-2" :messages="$errors->get('active')" />
```

Comment:
`<x-form-validation for="email" />` was already created as a component to handle validation errors.

Reply:
my understanding of this is that we should use <x-form-validation> in all cases instead of <x-input-error>?

Reply:
Did I input error already exist?

Reply:
I don't understand. I borrowed the <x-input-error> from elsewhere in the code, <x-form-validation> is present elsewhere. There are plenty of places where both are used, even beyond what I have touched.